src/main/scala/no/ndla/learningpathapi/validation/LearningPathValidator.scala

```diff
@@ -33,6 +36,17 @@ trait LearningPathValidator {
       }
     }
 
+    def validateTitle(lp: NewCopyLearningPathV2): NewCopyLearningPathV2 = {
+      val validationMessage = (lp.title.isEmpty, lp.language.isEmpty) match {
```

Comment:
Vi trenger vel tilsvarende validering for andre språkfelter som kan settes via `NewCopyLearningPath`?